src/content/app/entity-viewer/variant-view/variant-view-sidebar/overview/VariantOverview.tsx

```diff
@@ -78,7 +78,7 @@ const VariantOverview = () => {
 
   return (
     <div className={styles.overviewContainer}>
-      <section className={styles.defaultContent}>
+      <section className={styles.sectionContent}>
         <div>
           <span className={styles.labelTextStrong}>{variant.name}</span>
           <span className={styles.variantType}>
```

Comment:
_(sligntly below, where there is the VariantDB component)_

here is XD:
![image](https://github.com/Ensembl/ensembl-client/assets/6834224/9261e0b1-f629-48be-8380-a56024bd4beb)

here is the site:

![image](https://github.com/Ensembl/ensembl-client/assets/6834224/e5de0a44-5ea2-4689-b1d1-08ef2ee2479e)


Notice the font size of "Release 156"

Also, notice the font size of the same element in dev XD for genome browser:
https://xd.adobe.com/view/77e9782a-16ef-4a44-8cd8-41109abb35b5-22ba/

Looks like it should be made smaller?

Reply:
Will do this in a different PR as we discussed